content/id/index.md

```diff
@@ -1,22 +1,23 @@
 # JavaScript I/O
 
-Bringing [ES6](es6.html) to the Node Community!
+Membawa [ES6](es6.html) ke Komunitas Node!
 
-[io.js](https://github.com/iojs/io.js) is an [npm](https://www.npmjs.org/) compatible platform originally based on [node.js](https://nodejs.org/)&#8482;.
+[io.js](https://github.com/iojs/io.js) adalah [npm](https://www.npmjs.org/) platform yang aslinya kompatibel dengan [node.js] (https: //nodejs.org/)™.
 
 [![io.js](../images/1.0.0.png)](https://iojs.org/dist/v1.2.0/)
 
-[Version 1.2.0](https://iojs.org/dist/v1.2.0/)
+[Versi 1.2.0](https://iojs.org/dist/v1.2.0/)
 
-Download for
+
+Unduh untuk
 [Linux](https://iojs.org/dist/v1.2.0/iojs-v1.2.0-linux-x64.tar.xz),
 [Win32](https://iojs.org/dist/v1.2.0/iojs-v1.2.0-x86.msi), [Win64](https://iojs.org/dist/v1.2.0/iojs-v1.2.0-x64.msi),
 or
 [Mac](https://iojs.org/dist/v1.2.0/iojs-v1.2.0.pkg).
 
 
-[Changelog](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md)
+[Log Perubahan](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md)
 
-[Nightly releases](https://iojs.org/download/nightly/) are available for testing.
+[Relase setiap malam](https://iojs.org/download/nightly/) tersedia untuk pengujian.
```

Comment:
thanks for review, i think so, `open governance` more understand at moment, because `Pemerintahan terbuka` sounds of weird